extensions/wikia/PageShare/PageShareHelper.class.php

```diff
@@ -46,7 +47,11 @@ public static function getLangForPageShare( $browserLang, $useLang ) {
 		}
 	}
 
-	public static function isValidShareService( $service, $lang ) {
+	public static function isValidShareService( $service, $lang, $isTouchScreen ) {
+		//Don't display LINE social network (because it's a mobile app) on desktops
+		if ( $isTouchScreen === 'false' && ( $service['name'] === self::LINE_NAME ) ) {
```

Comment:
Then maybe you could use `$isTouchScreen = $this->request->getBool( 'isTouchScreen' );` from https://github.com/Wikia/app/blob/dev/includes/wikia/nirvana/WikiaRequest.class.php#L105? Any external opinion on that, @bognix?